### PR TITLE
More brain damage lines!

### PR DESCRIPTION
**Pull request by YakumoChen**
:cl: Yakumo Chen
tweak: The mentally challenged are now a little more... challenged.
/:cl:

### CODE REVIEW

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -58,6 +58,15 @@
...
         "u just did the world a little bit more sad place for someone",
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?"
+		"#mah vaeleds..."
+		"ittt dumshit furem wahreurs thet don;t ple whin aboot muh nastelgia fookin retaded GOOKS how dum r u dipshets goh tek a wak in movin treffec FUK U"
+		"VEEEBAAAAAAAAAAAAAAAAA!!!!"
+		"dis keel tactekul chares..."
+		"BREON ES ATMUS I DUN GODTA EXPLEN SHET"
+		"wut da fack is @pick(gas)>"
+		"wats da plen opatives?"
+		"OwO wuts dis?"
```
Comment (kevinz000) [inline strings/brain_damage_lines.json]: uWu

**Comment by kevinz000** [comment]
can we add "quikgrabig iz a featurd"?

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (octareenroon91) [inline strings/brain_damage_lines.json]: `@pick(gas)>` what or why is this one?

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -120,4 +129,15 @@
...
+    "gas": [
+        "cabon doxide",
+        "okygen",
+        "nidogin",
+        "plesmo",
+        "n20"
+		"wada vapurr"
+		"poron"
+		"fureeun"
+    ]
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: Bad json

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DoctoreMad** [comment]
>No "Me Unzips Peenees!"

Come on basic shitposting

**Comment by Kromgar** [comment]
whats the plan operatives? I ded

**Comment by dannno** [comment]
way too tryhard

**Comment by dannno** [comment]
WAAAAAAAAY too tryhard, these look terrible

**Comment by Bawhoppen** [comment]
Danno is right

**Comment by dannno** [comment]
they reek of a humorless oaf trying desperately to make a funny joke, but not understanding what makes the joke funny to begin with so it's all just vapid and cosmetic

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (ChangelingRain) [inline strings/brain_damage_lines.json]: Θώθ not OwO

File: strings/brain_damage_lines.json (hunk 2 above)
Comment (YakumoChen) [inline strings/brain_damage_lines.json]: I'm not a coder, please tell me how I'd fix thid

**Comment by YakumoChen** [comment]
You could say that about any brain damage lines

**Comment by YakumoChen** [comment]
Dunno is just salty because one of the lines is making fun of the toolboxes anyway

**Comment by dannno** [comment]
These honestly stand out among brain damage lines because you clearly tried way too hard and it reads like it

which line is supposed to be about toolboxes
???

**Comment by YakumoChen** [comment]
It's supposed to sound like gibbering retards spewing dumb memes, that's the joke.

The it dumbshit forum warriors line is the one you're looking for

**Comment by YakumoChen** [comment]
I could make the lines more clear but that wouldn't sound as retarded as I feel they deserve?

**Comment by dannno** [comment]
I guess you're right actually, you are a dumb retard spewing dumb memes, so perhaps this vitriol actually better captures the essence of "brain damage"

Maybe we were wrong and instead of funny game references we should make more brain damage lines be thinly veiled cheese and whine

tight win

**Comment by YakumoChen** [comment]
You sure seem butthurt about me dunno,coming in my pr just to attack me. Did I robust you too hard in a round once or something?

**Comment by kevinz000** [comment]
danno

**Comment by dannno** [comment]
jokes on you I don't play
"you seem butthurt" sure is a good way to deflect from the fact that you were clearly upset when you wrote some of these lines
don't delude yourself, I didn't even know it was supposed to be aimed at me until you told me it was
apply yourself nigga, this is garbage

**Comment by YakumoChen** [comment]
I added to a list of gibbering retard messages for shots and giggles. No need to project some kind of hateful tone into my pr. 

You're the only one getting upset about this. You might as well be filling the comments with giant gooftext LMAO

**Comment by YakumoChen** [comment]
I don't even care about the sprites any more I just thought that post was salty enough to be hilarious

**Comment by YakumoChen** [comment]
Like how mad can someone even get over arguing about a bunch of pixels? That shits hilarious

**Comment by dannno** [comment]
good question?

**Comment by dannno** [comment]
hey can you hear me

**Comment by dannno** [comment]
can you hear me from up there

**Comment by YakumoChen** [comment]
Itt dumbshit github warriors that don;t even play whine bout mine brain damage fucking retarded GOOKS go take a walk in moving traffic FUCK YOU

**Comment by dannno** [comment]
ok i definitely didn't write this

**Comment by dannno** [comment]
are you trying to attribute something sawrge said to me you dull fuck

**Comment by YakumoChen** [comment]
Didn't say you wrote it, it was just in your thread

**Comment by dannno** [comment]
you are a wastrel

**Comment by YakumoChen** [comment]
Is that a suggestion to add to the pr? :^)

**Comment by dannno** [comment]
you ever heard of kendo?

**Comment by YakumoChen** [comment]
Ignoring dunno, I think I'm going to add some Old Man Henderson quotes to the or when I get home, among other things.

**Comment by YakumoChen** [comment]
added a few more quotes, notably the OMH one I promised, more suggestions are welcome.

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (RemieRichards) [inline strings/brain_damage_lines.json]: odds are that won't render.

**Comment by Dorsisdwarf** [comment]
what the fuck is going on in this pr

idiocy aside I dont really like these, or feel like we need to keep the memes in the braindead moron-list updated

**Comment by RemieRichards** [comment]
@YakumoChen Check my commit to see what you did wrong.

**Comment by lzimann** [comment]
i'll be honest, don't really like these

**Comment by lzimann** [comment]
Anyway, if anyone wants to merge this, feel free to reopen and do it.

**Pull request closed without merging**